### DISCUSSION TITLE
Will Pulsar continue to support Windows 7 and Windows 8.1?

### DISCUSSION THREAD

**Discussion by ghost** [pulsar-edit/.github#256] [General]
https://www.electronjs.org/blog/windows-7-to-8-1-deprecation-notice

**Comment by confused-Techie** [comment]
Unfortunately, we do plan to upgrade to the latest Electron as soon as possible, in which case Electron itself will stop supporting these older Windows versions, and so will Pulsar because of that.

To be clear, we have been working hard for over a year now to move to the latest Electron, so don't expect this change tomorrow or next month, but eventually we will move to the latest Electron, at which point we have no say in the matter about supporting these older Windows versions. When we do make this change, it will be very obvious and it will not take you by surprise, as getting to latest Electron comes with several other large changes, so you'll be able to know when it happens very clearly.
